### PR TITLE
Model agnostic example

### PR DESCRIPTION
**Pull request by ssharpe42**
- Add model agnostic example notebook
- remove shap dependency
- update sklearn 1.0.1 -- > 1.1.0

### CODE REVIEW

File: ablation/baseline.py [hunk 1]
```diff
@@ -17,8 +16,12 @@
     opposite_class,
     training,
 )
+from .utils.general import sample
 from .utils.transform import le_to_ohe, ohe_to_le
 
+# from shap import sample
```
Comment (danielbarcklow) [inline ablation/baseline.py:22]: Is there a reason to keep this?

File: ablation/distributions.py [hunk 2]
```diff
@@ -8,10 +8,12 @@
 from numpy.random import permutation, randn
 from scipy import stats
 from scipy.ndimage import gaussian_filter
-from shap import sample
+
+# from shap import sample
```
Comment (danielbarcklow) [inline ablation/distributions.py:12]: Reason to keep this commented and not just remove it?

**Review by danielbarcklow — APPROVED** [review]
Just had the comments about commenting out imports instead of removing, but not formally requesting changes.

File: ablation/distributions.py (hunk 2 above)
Comment (ssharpe42) [reply ablation/distributions.py:12]: yes forgot thx

**Review by danielbarcklow — APPROVED** [review]

**Pull request merged**